src/main/java/uk/gov/hmcts/reform/blobrouter/data/reconciliation/reports/ReconciliationReportRowMapper.java

```diff
@@ -19,7 +19,8 @@ public ReconciliationReport mapRow(ResultSet rs, int rowNum) throws SQLException
             UUID.fromString(rs.getString("id")),
             UUID.fromString(rs.getString("envelope_supplier_statement_id")),
             rs.getString("account"),
-            rs.getString("content"),
+            rs.getString("summaryContent"),
+            rs.getString("detailedContent"),
```

Comment:
does this work?? column names have underscores

Reply:
it is draft. please review later.